src/footer/footer.macros.html

```diff
@@ -0,0 +1,59 @@
+{#
+  * @overview Macros for standard and simplified site Footers
+#}
+
+{% from '../icons/icons.macros.html' import icon as icon %}
+
+{#
+  * Renders a standard or simplified footer
+#}
+{% macro footer(simple=false, class=null, id=null) %}
+<footer class="footer{% if simple %} footer--simple{% endif %}{% if class %} {{ class }}{% endif %}"{% if id %} id="{{ id }}"{% endif %}>
+  <div class="footer-top r-inner">
+    {{ caller() }}
+  </div>
+  <div class="footer-bottom">
+    <div class="footer-nav r-inner">
+      <div class="footer-nav-group footer-nav-group--bottom">
+        <span class="footer-nav-country-icons">
+          <a class="footer-nav-group-link footer-nav-group-link--country" href="?country_id=49">
+            {{ icon('usa', size='small') }}
+          </a>
+          <a class="footer-nav-group-link footer-nav-group-link--country" href="?country_id=204">
+            {{ icon('canada', size='small') }}
+          </a>
+        </span>
+        <div class="footer-nav-group-text--mobile">
```

Comment:
@misscs Agree that the footer needs simplification. I'm gonna have to come back to it when I have more time after the landing fit andfinish.